frontend/src/js/tooltip/Tooltip.tsx

```diff
@@ -164,9 +171,23 @@ const Tooltip = () => {
         />
         <Head>
           <PinnedLabel>
-            <TypeIcon icon={isFolder ? faFolder : faMinus} />
+            <TypeIcon
+              icon={
+                // Parent is not element tooltipped or tooltip is a folder
+                isFolder || parent !== label
+                  ? // If the parent exists we show the folder icon
+                    parent
+                    ? faFolderOpen
```

Comment:
Hmm. So when parent is the same as label, we still show the `faFolderOpen`?  I don't think this makes sense. I'd vote to never show a `faFolderOpen`, but instead show a faFolder `regular` icon for the  concept tree "category" folders, to match the icons displayed in the tree. The attribute to check would be `const isTreeFolder = !parent.detailsAvailable`